jekyll/_cci2/contexts.md

```diff
@@ -142,7 +171,7 @@ You can combine several contexts for a single job by just adding them to the con
 ## Restricting a context
 {: #restricting-a-context }
 
-CircleCI enables you to restrict secret environment variables at run time by adding security groups to contexts. Only organization administrators may add *security groups* to a new or existing context. Security groups are your organization's GitHub teams. If you are using CircleCI Server with LDAP authentication, then LDAP groups also define security groups. After a security group is added to a context, only members of that security group who are also CircleCI users may access the context and use the associated environment variables.
+CircleCI enables you to restrict secret environment variables at run time by adding security groups to contexts. Only organization administrators may add *security groups* to a new or existing context. Security groups are your organization's GitHub teams. If you are using CircleCI server v2.xwith LDAP authentication, then LDAP groups also define security groups. After a security group is added to a context, only members of that security group who are also CircleCI users may access the context and use the associated environment variables.
```

Comment:
@rosieyohannan There appears to be a missing space  between "server v2.z" and "with" in the above. I'll approve the PR regardless, but wanted to point this out in case you wanted to take care of it in this PR instead of making a new one after.